.changeset/mean-coats-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-site": major
+---
+
+Enforce aspect ratio for `PixelImageBlock` and `Image`
```

Comment:
```suggestion
Make `aspectRatio` prop required for `PixelImageBlock` and `Image`
```

Reply:
I've extended the changeset with 460b6046b6bc7ba906a173f728033c1450e5d4a3.